content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -213,6 +213,9 @@ export class _DiscoveryStreamBase extends React.PureComponent {
             dispatch={this.props.dispatch}
             items={component.properties.items}
             cta_variant={component.cta_variant}
+            display_engagement_labels={
+              this.props.Prefs.values[`discoverystream.engagementLabelEnabled`]
```

Comment:
nit: preferably have the pref declared in the start  and use it here something like PREF_ENGAGEMENT = "discoverystream.engagementLabelEnabled";